####################################################################################################
This patch file is from Qualcomm Technologies, Inc. and is provided under the following license:

Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
SPDX-License-Identifier: BSD-3-Clause-Clear
####################################################################################################
From dcfa3177ae6bdd8911ddca570625ad14f1dc9589 Mon Sep 17 00:00:00 2001
From: Stilyan Uzunov <suzunov@qti.qualcomm.com>
Date: Thu, 22 May 2025 15:13:43 +0300
Subject: [PATCH] libdmabufheap: Update generated libs version

- Update generated libdmabufheap libs to match pkg-config version.

Upstream-Status: Pending
Signed-off-by: Stilyan Uzunov <suzunov@qti.qualcomm.com>
---
 Makefile.am  | 2 +-
 configure.ac | 3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 211dc58..d587e03 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -21,7 +21,7 @@ if USE_ION
 libdmabufheap_la_CPPFLAGS +=  -DUSE_ION
 endif
 libdmabufheap_la_CPPFLAGS +=  @LIBION_CFLAGS@
-libdmabufheap_la_LDFLAGS :=  @LIBION_LIBS@
+libdmabufheap_la_LDFLAGS :=  @LIBION_LIBS@ -version-info $(subst .,:, $(PACKAGE_VERSION))
 
 # Export headers
 dmabufheap_includedir=$(includedir)/BufferAllocator
diff --git a/configure.ac b/configure.ac
index f96dd56..806b6be 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ(2.61)
-AC_INIT([libdmabufheap],1.0.0)
+AC_INIT([libdmabufheap],0.0.0)
 AM_INIT_AUTOMAKE([-Wall gnu foreign])
 AC_CONFIG_SRCDIR([Makefile.am])
 AC_CONFIG_HEADERS([config.h])
@@ -40,4 +40,3 @@ AC_CONFIG_FILES([ \
     libdmabufheap.pc
 ])
 AC_OUTPUT
-
-- 
2.43.0

